### PR TITLE
Add sensors api data to the map

### PR DESCRIPTION
**Pull request by nerdlet**
## Description
#165307866

## Screenshots
image

## Tasks
Include specific tasks in the order they need to be done in. Include links to specific lines of code where the task should happen at.
- [x] Add new map to repository
- [x] Remove betterplace div
- [x] Add sensors hexbin layer to the new map

### CODE REVIEW

**Comment by nerdlet** [comment]
@kilemensi @karimkawambwa Ready for review

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "luft_node",
+  "name": "sensors.africa.feinstaub-map-v2",
```
Comment (kilemensi) [inline package-lock.json:2]: Changes should be done in `package.json`... `package-lock.json` is auto-generated when one uses `npm`

File: src/index.html [hunk 2]
```diff
@@ -1,162 +1,210 @@
...
+      #loading div {
+        position: absolute;
+        top: 40%;
+        width: 100%;
+        font-family: Arial, serif;
+        font-size: 200%;
+        font-weight: bold;
+      }
+    </style>
+    <title>Sensors.AFRICA BASE</title>
```
Comment (kilemensi) [inline src/index.html]: Why BASE? And why not just sensors.AFRICA?

**Review by kilemensi — COMMENTED** [review]
:+1:  @nerdlet:

-----------------------------------

1. Is it possible to use same editing style as the original `repo` to avoid unnecessary formatting changes?
2. Don't we need to define `config.js` that defaults to loading map using one of the cities we have sensors in or Africa at least rather than world map?
3. It's good habit to separate project setup changes (`chore`) such as adding GitHub Issue templates from adding sensors data (`feature`).

File: src/index.html (hunk 2 above)
Comment (nerdlet) [reply src/index.html]: @kilemensi That was meant to be a placeholder. Just removed it

**Comment by nerdlet** [comment]
@kilemensi Made the changes above and used a different editor for it. It should work
 1. Map default is on Africa
2. Increased zoom level

**Review by kilemensi — APPROVED** [review]
Lets deploy it @nerdlet :rocket:

**Pull request merged**